test/foundry/unit/TokenDistributor/TokenDistributor.t.sol

```diff
@@ -562,7 +562,7 @@ contract TokenDistributorTest is TokenDistributorSetup {
     }
 
     /// @notice test claimMany
-    function testClaimMany() public {
+    function testClaimManyOnce() public {
```

Comment:
Please add token balance assertions pre- and post-claim for user1 and the contract